### PR TITLE
Build the container using nix docker tools

### PR DESCRIPTION
**Pull request by TristanCacqueray**
This change removes the builder container so that Monocle uses the same toolchain for development and production.

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -38,9 +45,13 @@ jobs:
           CRAWLERS_API_KEY=secret
           GITHUB_TOKEN=123
           EOF
+      - name: Build monocle-exe container
+        run: "nix build .#containerMonocle; docker load < ./result"
+      - name: Build docker image
+        run: |
```
Comment (morucci) [inline .github/workflows/docker.yaml:51]: It seems we issue twice the same command

**Review by morucci — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (TristanCacqueray) [reply .github/workflows/docker.yaml:51]: What do you mean? The first docker load setup the "monocle-exe" image, and docker build produces the final "monocle" image, with the web assets.

Comment (morucci) [reply .github/workflows/docker.yaml:51]: https://github.com/change-metrics/monocle/blob/6099cfa475588a6497f90e3f952ad710949bee6c/.github/workflows/docker.yaml#L50-L55

Comment (TristanCacqueray) [reply .github/workflows/docker.yaml]: oh ok, then 8f9ca41 fix that :)

**Comment by morucci** [comment]
@TristanCacqueray I did a build image with nix, then run docker build and the devel demo deployment was a success.
So Let's merge \o/

**Comment by morucci** [comment]
Need to check if the master image is well published by the CI

**Pull request merged**